controllers/article-controllers.js

```diff
@@ -17,4 +27,22 @@ const getArticle = (request, response, next) => {
     });
 };
 
-module.exports = { getArticle, getAllArticles };
+const getArticleComments = (request, response, next) => {
+  const { article_id } = request.params;
+  const promises = [fetchArticleComments(article_id)];
+
+  if (article_id) {
+    promises.push(checkIdExists(article_id));
+  }
```

Comment:
You could essentially make use of a pre-existing model here, but this works fine too.